pages/belt_docs/list.mdx

```diff
@@ -17,64 +17,158 @@ type t('a) = list('a);
 let length: t('a) => int;
 ```
 
+```reason
+length(someList);
+```
+
+**Example**
+
+```reason
+Belt.List.length([1, 2, 3]); /* 3 */
+```
+
 ## size
 
 ```
 let size: t('a) => int;
 ```
 
+**See:** [length](#length)
+
 ## head
 
 ```
 let head: t('a) => option('a);
 ```
 
+```reason
+head(someList);
+```
+
+Returns `Some(value)` where `value` is the first element in the list, or `None` if `someList` is an empty list.
+
+**Examples**
+
+```reason
+Belt.List.head([]); /* None */
+
+Belt.List.head([1, 2, 3]); /* Some(1) */
+```
+
 ## headExn
 
 ```
 let headExn: t('a) => 'a;
 ```
 
+```reason
+head(someList);
+```
+
+Same as [head](#head), but raises an exception if `someList` is empty. Use with care.
```

Comment:
I wonder if we should show use-cases with exception handling somewhere.. probably not at every example, but maybe in a concepts overview?